src/main/kotlin/app/revanced/patches/all/interaction/gestures/patch/PredictiveBackGesturePatch.kt

```diff
@@ -8,26 +8,29 @@ import app.revanced.patcher.patch.PatchResult
 import app.revanced.patcher.patch.PatchResultSuccess
 import app.revanced.patcher.patch.ResourcePatch
 import app.revanced.patcher.patch.annotations.Patch
-import app.revanced.patches.shared.predictivebackgesture.annotations.PredictiveBackCompatibility
 
 @Patch
 @Name("predictive-back-gesture")
-@Description("Enables the predictive back gesture introduced in Android 13.")
-@PredictiveBackCompatibility
+@Description("Enables the predictive back gesture introduced on Android 13.")
 @Version("0.0.1")
 class PredictiveBackGesturePatch : ResourcePatch {
     override fun execute(context: ResourceContext): PatchResult {
         context.xmlEditor["AndroidManifest.xml"].use { editor ->
             val document = editor.file
-            val application = document.getElementsByTagName("application").item(0)
 
-            if(application.attributes.getNamedItem("android:enableOnBackInvokedCallback") == null) {
-                val attr = document.createAttribute("android:enableOnBackInvokedCallback")
-                attr.value = "true"
-                application.attributes.setNamedItem(attr)
+            with(document.getElementsByTagName("application").item(0)) {
+                attributes.getNamedItem(FLAG)?.let {
+                    document.createAttribute(FLAG)
+                        .apply { value = "true" }
+                        .let(attributes::setNamedItem)
+                }
```

Comment:
@oSumAtrIX This looks very incorrect. The previous code checked if `attributes.getNamedItem("android:enableOnBackInvokedCallback")` is `null` before creating the attribute, but your change instead creates the attribute **only if the attribute already exists** (`?.let` means "if not null, run this block")

Reply:
Its not very incorrect, but the logic should be inverted, please open a PR or issue.